.env.example

```diff
@@ -46,3 +46,6 @@ BANK_ACCOUNT_REQUEST_EMAIL_TARGET=help@doma.ai
 # APPLE_VOIP_PUSH_TOKEN_TEST=<TOKEN>
 
 MOBILE_APP_IDS = { "resident": { "ios": "ai.doma.clients", "android": "ai.doma.client" }, "master": { "ios": "ai.doma.master", "android": "ai.doma.master" }}
+
+DAILY_SAME_TICKET_LIMIT=2
```

Comment:
Seems like it's not necessary to build an app...

Reply:
We really can skip in here